test/e2e/kubevirt/net.go

```diff
@@ -39,6 +41,30 @@ func RetrieveCachedGatewayMAC(vmi *kubevirtv1.VirtualMachineInstance, dev, cidr
 	return outputSplit[4], nil
 }
 
+func RetrieveIPv6GatwayPaths(vmi *v1.VirtualMachineInstance) ([]string, error) {
```

Comment:
```suggestion
func RetrieveIPv6GatewayPaths(vmi *v1.VirtualMachineInstance) ([]string, error) {
```

Reply:
And actually, what do you think about naming this `RetrieveIPv6Gateways` ? The `path` suffix confuses me.